cost-analyzer/values-thanos.yaml

```diff
@@ -28,7 +28,7 @@ prometheus:
     enableAdminApi: true
     sidecarContainers:
     - name: thanos-sidecar
-      image: thanosio/thanos:v0.15.0
+      image: thanosio/thanos:v0.19.0
```

Comment:
Why just the sidecar here?

Reply:
Ok, nevermind - I see that sidecar didn't get bumped in the previous update.

Reply:
Yes. Two reasons:

Sidecar interoperates with prometheus closely, so prometheus being significantly behind sidecar seemed bad

Users don't upgrade all clusters simultaneously anyway, so we needed to support some level of interoperability anyway between 0.15 and 0.19.